src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/ApplicationModuleDeployer.java

```diff
@@ -163,6 +164,23 @@ public void run() {
 		return moduleDeploymentId;
 	}
 
+	private int deduceMemory(Map<String, String> deploymentProperties) {
+		String override = deploymentProperties.get("cloudfoundry.memory");
```

Comment:
Kind of a nitpick, but we may want to start using constants at this point, just to be able to get a more centralized view of the supported options.